### PR TITLE
SVS recipe for the JSUT-song corpus

### PR DESCRIPTION
**Pull request by HANJionghao**
## What?

Added a singing voice synthesis recipe for JSUT-song corpus under egs2/jsut_song/svs1.

## Why?

Create a singing voice synthesis recipe for JSUT-song corpus.

## See also

https://github.com/espnet/espnet/blob/master/egs2/nit_song070/svs1/

### CODE REVIEW

File: egs2/jsut_song/svs1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+eval_set="eval"
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    log "stage 0: Data Download"
+    # The jsut-song data should be downloaded from https://sites.google.com/site/shinnosuketakamichi/publication/jsut-song
+    # Terms from https://sites.google.com/site/shinnosuketakamichi/publication/jsut-song
+
+    # Please ensure that you've downloaded songs (jsut-song_ver1.zip) and labels (jsut-song_label.zip) to ${JSUT_SONG}, and unzip them before proceeding
+    # unzip -o ${JSUT_SONG}/*.zip
+    # rm ${JSUT_SONG}/*.zip
```
Comment (ftshijt) [inline egs2/jsut_song/svs1/local/data.sh:47]:
```suggestion
    unzip -o ${JSUT_SONG}/*.zip
    rm ${JSUT_SONG}/*.zip
```

File: egs2/jsut_song/svs1/run.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+# text related processing arguments
+g2p=none
+cleaner=none
+
+./svs.sh \
+    --lang jp \
+    --stage 1 \
```
Comment (ftshijt) [inline egs2/jsut_song/svs1/run.sh:38]:
```suggestion
```

File: egs2/jsut_song/svs1/run.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    --local_data_opts "--stage 0 $(pwd)" \
+    --feats_type raw \
+    --pitch_extract None \
```
Comment (ftshijt) [inline egs2/jsut_song/svs1/run.sh:41]:
```suggestion
```

File: egs2/jsut_song/svs1/run.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+inference_config=conf/decode.yaml
+
+# text related processing arguments
+g2p=none
+cleaner=none
+
+./svs.sh \
+    --lang jp \
+    --stage 1 \
+    --local_data_opts "--stage 0 $(pwd)" \
```
Comment (ftshijt) [inline egs2/jsut_song/svs1/run.sh:39]: Is the pwd used?

File: egs2/jsut_song/svs1/run.sh [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    --train_config "${train_config}" \
+    --inference_config "${inference_config}" \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    --score_feats_extract "${score_feats_extract}" \
+    --srctexts "data/${train_set}/text" \
+    --ngpu 1 \
+    --min_wav_duration 0.1 \
+    --max_wav_duration 90 \
```
Comment (ftshijt) [inline egs2/jsut_song/svs1/run.sh:60]: Is default not working in the case?

File: egs2/jsut_song/svs1/run.sh [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    --cleaner ${cleaner} \
+    --train_config "${train_config}" \
+    --inference_config "${inference_config}" \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    --score_feats_extract "${score_feats_extract}" \
+    --srctexts "data/${train_set}/text" \
+    --ngpu 1 \
+    --min_wav_duration 0.1 \
```
Comment (ftshijt) [inline egs2/jsut_song/svs1/run.sh:59]:
```suggestion
```

**Review by ftshijt — COMMENTED** [review]
Thanks for the contribution! Please also add an entry in egs2/README.md for the recipe.

File: egs2/jsut_song/svs1/run.sh (hunk 4 above)
Comment (HANJionghao) [reply egs2/jsut_song/svs1/run.sh:38]: No. It can be deleted

File: egs2/jsut_song/svs1/run.sh (hunk 5 above)
Comment (HANJionghao) [reply egs2/jsut_song/svs1/run.sh]: Removed --min_wav_duration 0.1 and --max_wav_duration 90 in the latest commit

**Comment by ftshijt** [comment]
Thanks for your effort. I believe the current repo is ready to merge. will merge it after the CI issues are fixed.

**Comment by ftshijt** [comment]
@HANJionghao Please fix the CI issue~

**Comment by ftshijt** [comment]
Thanks for the update!

**Pull request merged**